documents/charter.md

```diff
@@ -1,23 +1,24 @@
 
 Video traffic usually employs adaptive bitrate (ABR) schemes to dynamically adjust the video quality (and thus the data rate) in response to changing network conditions.
-In the presence of traffic shaping, the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper.
+In the presence of a network operator performing traffic shaping (also known as throttling), the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper.
 In practice this is extremely difficult to achieve while maintaining a good user experience.
 Application providers are even designing algorithms to detect the presence of such traffic shapers and estimate the targeted shaping rate, however, these algorithms are likely to be both inaccurate and complex.
 Instead, it would be beneficial, for both the application provider and network operator, to signal the shaper rate to the application to self-adapt their video traffic to conform to the specified characteristics.
-The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
+The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback. Alternatively, networks could also cease traffic shaping of applications like ABR video streaming and, in addition, could further invest in network capacity. A final alternative is that application developers could detect that traffic was traversing a mobile network link and request a lower bitrate stream from application servers. 
 
 The Secure Communication of Network Properties (SCONEPRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video. 
 - The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video delivered from a server to a client, using QUIC connections carrying the application signaling traffic.
 - Work to support TCP or other transport protocols may be considered later in the working group, however, these considerations shouldn't distract from support for video over QUIC. 
 - Further use cases may be considered later in the working group, however, it is not assumed that future use cases must or can be addressed by the same protocol. In essence, any protocol specified by the working group should be tailored to solve a specific use case.
+- The working group will also document how (1) providers might alternatively choose to invest in network capacity, (2) what the effect of not using rate shaping on mobile networks would have, and (3) how application providers on mobile devices could detect the use of the mobile interface and select a lower bitrate as a result. 
```

Comment:
> (1) I don't think the group should document how providers should scale their capacity, since that is a well-understood business topic and involves numerous non-technical considerations. 

I am not suggesting a long description of how capacity can be added. Rather, I would like to see text that says that capacity is one way to solve this problem (in whole or part).

Reply:
Oh sure, agree.